doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -33,6 +33,7 @@
 - Removed parameter `pimcore.admin.session.attribute_bags`
 - TargetingSessionBagListener - changed the signature of `__construct`.
 - Removed deprecated `Pimcore\Db\ConnectionInterface` interface, `Pimcore\Db\Connection` class and `Pimcore\Db\PimcoreExtensionsTrait` trait.
+- Removed deprecated `Pimcore\Model\Element\Service::getSaveCopyName()` method, please use the `Pimcore\Model\Element\Service::getSafeCopyName()` method instead.
```

Comment:
[Docs ](https://github.com/pimcore/pimcore/blob/bc483848fba6f3c937676361d634ecedb9d0247c/doc/Development_Documentation/19_Development_Tools_and_Details/17_Static_Helpers.md#element-service)should be updated as well (better on 10.5).

Reply:
@dvesh3 I created a PR for 10.5 https://github.com/pimcore/pimcore/pull/14247